### PR TITLE
Full functionality for Calendar Page

### PR DESCRIPTION
**Pull request by maitmd**
Everything from, adding, removing, completing should be done for this. Didn't know any better way for the individual event menus so if there is a better alternative please let me know.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/kaylebredner/Project_CS372/issues"
+  },
+  "homepage": "https://github.com/kaylebredner/Project_CS372#readme",
+  "dependencies": {
+    "body-parser": "^1.20.1",
+    "express": "^4.18.2",
+    "https": "^1.0.0",
```
Comment (tarunJeevan) [inline package.json:23]: I thought https and path were built in modules?

**Review by tarunJeevan — APPROVED** [review]
Looks good. Should we create a main-copy branch to merge our branches into first? Like we did with the Capstone project?

File: package.json (hunk 1 above)
Comment (maitmd) [reply package.json:23]: I'm not sure. I included them in the npm install just in case.

**Comment by maitmd** [comment]
> Looks good. Should we create a main-copy branch to merge our branches into first? Like we did with the Capstone project?


Definitely not necessary here, that environment is just extremely volatile where this one is very clear cut.

**Review by AlexBishop01 — APPROVED** [review]
Looks great to me!

**Review by kaylebredner — APPROVED** [review]
looks good to me

**Pull request merged**